lib/mix/tasks/export.attendees.entries.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Mix.Tasks.Export.Attendees.Entries do
+  use Mix.Task
+
+  def run(_) do
+    Mix.Task.run("app.start")
+
+    IO.puts("uuid,name,email,entries")
+
+    Enum.each(
+      Safira.Accounts.list_active_attendees(),
+      fn a ->
+        IO.puts(csv_io(a))
+      end
+    )
+  end
```

Comment:
I think you could just:
```Elixir
Safira.Accounts.list_active_attendees()
|> Enum.each(fn a -> csv_io(a) |> Mix.shell().info end)
```

Reply:
I understand your tip, nice one. Thx 👍
About IO.puts vs Mix.shell().info, I just tried to make that code consistent when compared to other tasks to export information(e.g., export.attendees.ex). Anyway, mix.shell().info has been implemented on this task